src/views/recording.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect } from "react";
+import { connect, ConnectedProps } from "react-redux";
+import { RouteComponentProps, withRouter } from "react-router-dom";
+import { isTest } from "ui/utils/environment";
+import { validateUUID } from "ui/utils/helpers";
+import { setExpectedError } from "ui/actions/session";
+import { useIsRecordingInitialized, useGetRecordingOwnerUserId } from "ui/hooks/recordings";
+import BlankScreen from "ui/components/shared/BlankScreen";
+import { bootstrapApp } from "ui/setup";
+import { registerRecording } from "ui/utils/telemetry";
+import Upload from "views/upload";
+import DevTools from "ui/components/DevTools";
+
+import "ui/setup/dynamic/devtools";
+
+bootstrapApp();
```

Comment:
This is already called in `app.tsx` (and should be called there because it's also needed by the `Account` component)

Reply:
I think this was a copy/paste fail. It should be removed.